CAIPs/caip-25.md

```diff
@@ -7,7 +7,7 @@ status: Draft
 type: Standard
 created: 2020-10-14
 updated: 2022-10-26
-requires: ["2", "10"]
+requires: 2, 10
```

Comment:
It appears the `"`s were unnecessary, but the `[]`s *were* necessary.  Not sure if this is all Yaml, or the specific version of Yaml Jekyll uses, or just Jekyll, but:

https://talk.jekyllrb.com/t/solved-front-matter-array-with-multiple-values/4426

Reply:
OK then the other way around - but for now we had both styles which was not good:

CAIPs/CAIPs/caip-25.md:requires: ["2", "10"]
CAIPs/CAIPs/caip-76.md:requires: 10, 75

Reply:
Fixed!

Reply:
And a comment added to CAIP-1 and to the template.md